vm/src/obj/objstr.rs

```diff
@@ -413,12 +414,12 @@ impl PyString {
         for c in self.value.chars() {
             if c.is_lowercase() {
                 if !previous_is_cased {
-                    title.extend(c.to_uppercase());
+                    title.extend(c.to_titlecase());
```

Comment:
How can a single character be title cases? I guess something else is meant than python's titlecase function?

Reply:
This surprised me too, but there indeed exist single codepoint titlecase characters (I realized this from reading the cpython tests for `title` / `istitle`). [Some examples](https://github.com/OddCoincidence/unicode-casing/blob/master/src/lib.rs#L104-L105).

Reply:
"Latin Capital Letter D with Small Letter Z with Caron". Unicode is never boring! :)